drivers/usb/device/usb_dc_stm32.c

```diff
@@ -333,7 +422,7 @@ static int usb_dc_stm32_clock_disable(void)
 		LOG_ERR("Unable to disable USB clock");
 		return -EIO;
 	}
-#if DT_HAS_COMPAT_STATUS_OKAY(st_stm32_otghs) && defined(CONFIG_SOC_SERIES_STM32U5X)
+#if DT_HAS_COMPAT_STATUS_OKAY(st_stm32u5_otghs_phy)
 	LL_AHB2_GRP1_DisableClock(LL_AHB2_GRP1_PERIPH_USBPHY);
 #endif
```

Comment:
Isn't this a `clock_control_off(clk, (clock_control_subsys_t)&phy_pclken[0]);` now?

Reply:
Yes, it seems so. Should I replace it with `clock_control_off()` API?

Reply:
it would be nice.

Reply:
Done!